infrastructure/lib/addons/fluxv2.ts

```diff
@@ -127,6 +129,30 @@ export class FluxV2 extends Construct {
   });
   kustomizationManifest.node.addDependency(chart);
 
+  if (props.repoAddonPaths) {
+    for (var repoAddonPath of props.repoAddonPaths) {
+      const addonKustomizationManifest = props.cluster.addManifest('AddonKustomizationSelf', {
```

Comment:
could we wrap this in `function` as we are using it multiple times ?

Reply:
That's possible.
https://github.com/awslabs/kubernetes-iteration-toolkit/blob/da36db42fe000ace00c9396fb3abd0af226da3b0/infrastructure/lib/addons/fluxv2.ts#L110-L128
and https://github.com/awslabs/kubernetes-iteration-toolkit/blob/da36db42fe000ace00c9396fb3abd0af226da3b0/infrastructure/lib/addons/fluxv2.ts#L150-L168 are doing similar things for `Repo` and `testRepo`. We should clean them up as well eventually.